Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
They are still harcoded, but you can also inject them. Either change your commit message to reflect that, or find a right place to make a call to `setFormats`… but maybe all this would be easier if this was not static ?